library/src/main/scala/scala/scalajs/js/Date.scala

```diff
@@ -50,6 +50,12 @@ class Date extends js.Object {
 
   override def valueOf(): Double = js.native
 
+  /** Returns the numeric value of the specified date as the number of
+    * milliseconds since January 1, 1970, 00:00:00 UTC.
+    * (Negative values are returned for prior times).
+    *
+    * MDN
+    */
```

Comment:
Please follow the coding style for Scaladoc comments in this file:
```suggestion
  /**
   * Returns the numeric value of the specified date as the number of
   * milliseconds since January 1, 1970, 00:00:00 UTC.
   * (Negative values are returned for prior times).
   *
   * MDN
   */
```

Reply:
I actually had it this way as at first it's consistent with the other comments there but https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#scaladoc says it should start with the first line.

Reply:
Ah yes, if you follow that coding style you also find an acceptable way of formatting Scaladoc, but notice that the stars are also aligned differently from your code. Following that style, it would be:
```suggestion
  /** Returns the numeric value of the specified date as the number of
   *  milliseconds since January 1, 1970, 00:00:00 UTC.
   *  (Negative values are returned for prior times).
   *
   *  MDN
   */
```

Reply:
I went with your original suggestion for consistency.